src/theory/arith/arith_utilities.cpp

```diff
@@ -17,6 +17,9 @@
 
 #include <cmath>
 
+#include "theory/bv/theory_bv_utils.h"
```

Comment:
Can this be avoided? What do you use from here, and does it make sense to move that into some place else? If it is not a big effort, it might worth it for better separation between the theories.

Reply:
We've talked about separating generic utilities out from the specific theories in the past. At the moment, we haven't done this, e.g. many theories use generic arith utils.  For now, I think it makes sense to depend on bv utils here.

Reply:
Heres the issue I remembered: https://github.com/cvc5/cvc5-wishues/issues/113.